libs/components/toast/src/lib/modules/toast/toaster.component.ts

```diff
@@ -36,7 +37,15 @@ import { SkyToastDisplayDirection } from './types/toast-display-direction';
   selector: 'sky-toaster',
   templateUrl: './toaster.component.html',
   styleUrls: ['./toaster.component.scss'],
-  providers: [SkyToastAdapterService, SkyToasterService],
+  providers: [
+    SkyToastAdapterService,
+    SkyToasterService,
+    {
+      provide: SkyStackingContextService,
+      // Match z-index set in libs/components/theme/src/lib/styles/_public-api/_compat/_variables.scss
+      useValue: new SkyStackingContextService({ zIndex: 1001 }),
```

Comment:
Same comment here, as for the flyout.

Reply:
Switched to use an observable, added a comment to the scss variables file.